lib/openxml/drawingml/properties/path.rb

```diff
@@ -0,0 +1,49 @@
+# require "openxml/drawingml/properties/path_close"
+# require "openxml/drawingml/properties/path_move_to"
+# require "openxml/drawingml/properties/path_line_to"
+# require "openxml/drawingml/properties/path_arc_to"
+# require "openxml/drawingml/properties/path_quadratic_bezier_to"
+# require "openxml/drawingml/properties/path_cubic_bezier_to"
```

Comment:
What are the commented-out requires for?

Reply:
These child classes are not implemented yet (and for our immediate purposes, didn't need to be). I liked keeping the requires and the actual `child_classes` declaration (below), though, as a reference for implementing these in the future.

Reply:
👌